testsuite/integration/basic/src/test/java/org/wildfly/test/integration/jsp/JspELTestCase.java

```diff
@@ -120,14 +121,14 @@ public void testJavaLangImplicitClassELEvaluation31(@ArquillianResource URL url)
 
     private void commonTestPart(final URL url, final String possibleCausingIssues) throws Exception {
         final String responseBody = HttpRequest.get(url + "index.jsp", 10, TimeUnit.SECONDS);
-        Assert.assertTrue("Unexpected EL evaluation for ${Boolean.TRUE}; " + possibleCausingIssues,
-                responseBody.contains("Boolean.TRUE: --- " + Boolean.TRUE + " ---"));
-        Assert.assertTrue("Unexpected EL evaluation for ${Integer.MAX_VALUE}; " + possibleCausingIssues,
-                responseBody.contains("Integer.MAX_VALUE: --- " + Integer.MAX_VALUE + " ---"));
-        Assert.assertTrue("Unexpected EL evaluation for ${DummyConstants.FOO}; " + possibleCausingIssues,
-                responseBody.contains("DummyConstants.FOO: --- " + DummyConstants.FOO + " ---"));
-        Assert.assertTrue("Unexpected EL evaluation for ${DummyEnum.VALUE}; " + possibleCausingIssues,
-                responseBody.contains("DummyEnum.VALUE: --- " + DummyEnum.VALUE + " ---"));
+        Assert.assertThat("Unexpected EL evaluation for ${Boolean.TRUE}; " + possibleCausingIssues, responseBody,
+                CoreMatchers.containsString("Boolean.TRUE: --- " + Boolean.TRUE + " ---"));
+        Assert.assertThat("Unexpected EL evaluation for ${Integer.MAX_VALUE}; " + possibleCausingIssues, responseBody,
+                CoreMatchers.containsString("Integer.MAX_VALUE: --- " + Integer.MAX_VALUE + " ---"));
+        Assert.assertThat("Unexpected EL evaluation for ${DummyConstants.FOO};  " + possibleCausingIssues, responseBody,
+                CoreMatchers.containsString("DummyConstants.FOO: --- " + DummyConstants.FOO + " ---"));
+        Assert.assertThat("Unexpected EL evaluation for ${DummyEnum.VALUE}; " + possibleCausingIssues, responseBody,
+                CoreMatchers.containsString("DummyEnum.VALUE: --- " + DummyEnum.VALUE + " ---"));
```

Comment:
Is there a reason for this change? It looks like just the addition of using `CoreMatchers` instead of `String.contains()`.